Postman/Digital Excellence API.postman_collection.json

```diff
@@ -0,0 +1,800 @@
+{
+	"info": {
+		"_postman_id": "00e58bc3-a4d6-4c62-a484-a3f70dc823a3",
+		"name": "Digital Excellence API",
```

Comment:
To avoid issues with spaces, best to rename the file to Digital-Excellence-API. Change both the filename and name of collection